Topics/Teamwork.md

```diff
@@ -1 +1,15 @@
 ## Resources for Teamwork
+
+
+### Using Kanban Boards
+
+Whether its GitHub Projects, Trello, Asana or Jira, if you're working with a software development team, there's a good chance you will be working with a Kanban board for your project management.
+
+Using them effectively as a team is a combination of keeping things clear for yourself, and clear for others.
+
+Kanban boards are different from a personal todo-list in that they are shared with a team and updated all the time. This all-encompassing nature has the advantage that anyone can check the board and see what is being worked on and at what stage, but has the pre-requisite that of clear wording and being constantly updated to ensure clarity. Setting [SMART goals](https://www.atlassian.com/blog/productivity/how-to-write-smart-goals) become essential, ensuring everyone else has a clear idea of what you're working on.
```

Comment:
replace 'todo-list' with 'to-do list'
replace 'pre-requisite that' with 'prerequisite'

Reply:
Thanks! Fixed terms